source/reference/limitations.txt

```diff
@@ -132,10 +132,13 @@ Reversing
 Multiple Clusters
 -----------------
 
-- Syncing multiple source clusters to one destination cluster isn't
-  supported.
-- Syncing one source cluster to many destination clusters isn't
-  supported.
+- ``mongosync`` does not support syncing multiple source clusters to one
+  destination cluster. 
+- One ``mongosync`` instance cannot sync a source cluster to many
```

Comment:
Overall this LGTM, however -- do we want to specify here that you *can* run multiple mongosyncs from the same source cluster?

Reply:
I think it makes sense to specify! I added a note under the limitation